internal/entities/handlers/handler.go

```diff
@@ -102,7 +102,7 @@ func (b *handleEntityAndDoBase) handleRefreshEntityAndDo(msg *watermill.Message)
 		l.Debug().Msg("entity not retrieved")
 	}
 
-	err = b.forwardEntityCheck(ctx, ewp)
+	err = b.forwardEntityCheck(ctx, entMsg, ewp)
```

Comment:
Below here we could check if the properties don't match (via a defined error, mentioned below), and not log an error, since it isn't one.

Reply:
done!